technical-operations/middleware/supported-environments.md

```diff
@@ -17,9 +17,9 @@ After reading this, you can prepare a system where the Middleware will run.
 | **Operating System:** <br /> Windows        | **Windows 7** or higher <br />**.NET Framework >= 4.8** |
 | **Operating System:** <br /> Linux & macOS  | Ubuntu or Debian recommended<br />**Mono >= 6.8.0**<br />SOAP is not supported, gRPC is recommended |
 | **Operating System:** <br /> Android        | **Germany only**<br />**Android 7** or higher<br />Swissbit or Fiskaly TSSs only |
-| Databases                         | **SQLite** (local DB)<br />**MS SQL Server** (All versions currently supported by Microsoft)<br />**MySQL** (Germany only) |
-| Hardware                          | At least 200MB of memory per Queue<br />Multicore CPU recommended<br />The actual memory amount depends on the amount of receipts to process. |
-| Local Storage                     | At least 500 MB (200 MB for the Middleware + 200 MB reserved for update packages)<br />Optional: Storage for SQLite DB (~ 8-10Kb/receipt) |
+| **Databases**                         | **SQLite** (local DB)<br />**MS SQL Server** (All versions currently supported by Microsoft)<br />**MySQL** (Germany only) |
+| **Hardware**                          | At least 200MB of memory per Queue<br />Multicore CPU recommended <br /> ARM usable with restrictions - [more details here](https://github.com/fiskaltrust/interface-doc/tree/master/doc/middleware-de-kassensichv/operation-modes) <br />The actual memory amount depends on the amount of receipts to process. |

```

Comment:
Let's maybe add a separate row with "CPU Architecutre" and have:
- x86
- x64
- ARM (...the text with the link)